client/styles/components/_preferences.scss

```diff
@@ -105,6 +129,59 @@
 	width: #{44 / $base-font-size}rem;
 }
 
+.preference__subheading1 {
```

Comment:
if you use a SCSS placeholder, you can define a style for all of the tabs, and then each individual tab can extend this placeholder, but then have its own styles. very, very similar to object-oriented programming in java or c++.